### PR TITLE
Add leaflet.pm@0.12.2 w/ git auto-update

### LINKED ISSUE #9799: [Request] Add leaflet.pm
**Library name:** leaflet.pm
**Git repository url:** https://github.com/codeofsumit/leaflet.pm
**npm package url(optional):** 
**License(s):** ISC
**Official homepage:** https://codeofsumit.github.io/leaflet.pm/
**Wanna say something? Leave message here:** cc #6746

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by lcd78706**
Pull request for issue: #9799 
Related issue(s): # #9414 

@kennynaoh  Please review this PR. Thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/codeofsumit/leaflet.pm
 * Official website (optional, not the repository): http://codeofsumit.github.io/leaflet.pm/ 
 * NPM package url (optional): https://www.npmjs.com/package/leaflet.pm
 * GitHub / Bitbucket popularity (required): 
   - Count of stars: 33
   - Count of watchers: 5
   - Count of forks: 8
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 21
   - Downloads in the last month: 157

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #9414, close #9799, cc @codeofsumit

### CODE REVIEW

File: ajax/libs/leaflet.pm/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "type": "git",
+    "url": "git+https://github.com/codeofsumit/leaflet.pm.git"
+  },
+  "author": {
+    "name": "Sumit Kumar",
+    "email": "sk@outlook.com",
+    "url": "http://twitter.com/TweetsOfSumit"
+  },
+  "license": "ISC",
+  "homepage": "http://codeofsumit.github.io/leaflet.pm/",
```
Comment (kennynaoh) [inline ajax/libs/leaflet.pm/package.json]: It's better to use `https` here.

Comment (codeofsumit) [inline ajax/libs/leaflet.pm/package.json]: I'll change that as well.

**Comment by codeofsumit** [comment]
hi there, I'm the author of leaflet.pm. If there are issues with the library that I should fix in order to comply with this PR, please open an issue here: https://github.com/codeofsumit/leaflet.pm/issues
I'll have them fixed as soon as possible.

**Comment by PeterDaveHello** [comment]
@codeofsumit , thanks, if there is any problem, we'll open issue there.

**Comment by lcd78706** [comment]
@kennynaoh 
I have fixed the problem.

**Comment by kennynaoh** [comment]
@lcd78706 The `homepage` field is still wrong.

**Comment by lcd78706** [comment]
@kennynaoh 
I have fixed the problem.

**Comment by kennynaoh** [comment]
@lcd78706 You should wait for CI result before you tag me to review next time.

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**